### DISCUSSION TITLE
FXGL on Raspberry Pi

### DISCUSSION THREAD

**Discussion by AlmasB** [AlmasB/FXGL#1073] [Ideas]
This is the place for all discussions related to optimising FXGL for the Raspberry Pi.

@FDelporte last time we tested FXGL we hit a very odd issue with a 30fps cap for any scene graph change. Any updates since then?

**Comment by FDelporte** [comment]
Good question, need to retest with latest FXGL and JavaFX 17. Will do asap and report here about the results.
